pkg/kubelet/dockershim/docker_sandbox.go

```diff
@@ -199,6 +204,9 @@ func makeSandboxDockerConfig(c *runtimeApi.PodSandboxConfig, image string) *dock
 	labels := makeLabels(c.GetLabels(), c.GetAnnotations())
 	// Apply a label to distinguish sandboxes from regular containers.
 	labels[containerTypeLabelKey] = containerTypeLabelSandbox
+	// Apply a container name label for infra container. This is used in summary api.
+	// TODO(random-liu): Deprecate this label once container metrics is directly got from CRI.
+	labels[types.KubernetesContainerNameLabel] = dockertools.PodInfraContainerName
```

Comment:
Don't refer `dockertools.PodInfraContainerName` directly in all the files. It'd be a hassle to change all of them.
Suggestion:
1. In `naming.go`, set `sandboxContainerName`  to `leaky.PodInfraContainerName`, and add comments there.
2. Use only `sandboxContainerName` in the rest of the files in the docekrshim package.